ruff/__main__.py

```diff
@@ -0,0 +1,7 @@
+import os
+import sys
+import sysconfig
+
+if __name__ == "__main__":
+    ruff = os.path.join(sysconfig.get_path("scripts"), "ruff")
+    os.spawnv(os.P_WAIT, ruff, [ruff, *sys.argv[1:]])
```

Comment:
(Changed from `os.execv` because that seems to put the process in the background on Windows: python/cpython#53394.)